f2/src/App.js

```diff
@@ -62,18 +62,19 @@ const App = () => {
             bg="gray.50"
           >
             <Home />
-          </Stack>
 
-          <Layout>
-            <P fontSize="sm" my={3}>
+            <Layout>
+
+            <P fontSize="sm" my={3} aria-label="application version">
```

Comment:
@amazingphilippe  let me know if this small addition/hack to aria-label here will cause any issue. I'm trying to avoid refactor P at this point to pass aria attr as props :)